src/app/modals/syncing/syncing.component.html

```diff
@@ -1,45 +1,51 @@
-<div *ngIf="syncPercentage">
-  <div class="container">
-    <div class="content">
+<div class="container">
+  <div class="content">
+    <div class="section wallet-syncing">
+      <div class="title">Wallet is syncing</div>
+      <p>
+        Your wallet automatically synchronizes with the Particl network after a connection is established, but this
```

Comment:
From the description I though the modals should be removed ?

Reply:
We talked about this (on meeting) and decided it would be best to leave it there. Since you can't really fully use the wallet when it's syncing, it makes sense. If you don't like it, you can still close it and you'll see progress in the sidebar. If needed, by clicking the sync progress in sidebar, you can fire the sync modal again.

Reply:
alright then, let's merge this and see later how we integrate the new filtertransactions API and Transaction model